peerconnection.go

```diff
@@ -1138,7 +1138,7 @@ func (pc *PeerConnection) AddTrack(track *Track) (*RTPSender, error) {
 	var transceiver *RTPTransceiver
 	for _, t := range pc.rtpTransceivers {
 		if !t.stopped &&
-			// t.Sender == nil && // TODO: check that the sender has never sent
+			t.Sender != nil && // TODO: check that the sender has never sent
```

Comment:
Regarding this TODO - I think, a helper can be added and used here:

```go
func (r *RTPSender) sent() bool {
  r.mu.Lock()
  defer r.mu.Unlock()

  select {
    case <-r.sendCalled:
      return true
    default:
       return false
  }
}
```

Looks like `sendCalled` chan is made in constructor and closed on `.Send` call:
https://github.com/pions/webrtc/blob/f46ebf7effae3f373aa8fe8e0aa7c279572a49ad/rtpsender.go#L55-L81